packages/liquidator/src/liquidationStrategy.js

```diff
@@ -221,11 +229,7 @@ module.exports = (
     )
       return false;
     // liveness timer on withdraw has not passed time
-    if (
-      withdrawProgressPercent(withdrawLiveness, position.withdrawalRequestPassTimestamp, currentBlockTime) <
-      parseFloat(defenseActivationPercent)
-    )
-      return false;
+    if (!passedDefenseActivationPercent({ position, currentBlockTime })) return false;
```

Comment:
can you not just return `passedDefenseActivationPercent({ position, currentBlockTime })` at this point?

Reply:
true!